mmdet3d/core/visualizer/show_result.py

```diff
@@ -110,3 +110,64 @@ def show_result(points, gt_bboxes, pred_bboxes, out_dir, filename, show=True):
         pred_bboxes[:, 6] *= -1
         _write_oriented_bbox(pred_bboxes,
                              osp.join(result_path, f'{filename}_pred.ply'))
+
+
+def show_multi_modality_result(img,
+                               gt_bboxes,
+                               pred_bboxes,
+                               proj_mat,
+                               out_dir,
+                               filename,
+                               depth_bbox=False,
+                               img_metas=None,
+                               show=True):
+    """Convert multi-modality detection results into 2D results.
+
+    Project the predicted 3D bbox to 2D image plane and visualize them.
+
+    Args:
+        img (np.ndarray): The numpy array of image in cv2 fashion.
+        gt_bboxes (np.ndarray): Ground truth boxes.
+        pred_bboxes (np.ndarray): Predicted boxes.
+        proj_mat (numpy.array, shape=[4, 4]): The projection matrix
+            according to the camera intrinsic parameters.
+        out_dir (str): Path of output directory
+        filename (str): Filename of the current frame.
+        depth_bbox (bool): Whether we are projecting camera bbox or lidar bbox.
+        img_metas (dict): Used in projecting cameta bbox.
+        show (bool): Visualize the results online. Defaults to True.
+    """
+    if depth_bbox:
+        from .open3d_vis import draw_depth_bbox3d_on_img as draw_bbox
+    else:
+        from .open3d_vis import draw_lidar_bbox3d_on_img as draw_bbox
+
+    result_path = osp.join(out_dir, filename)
+    mmcv.mkdir_or_exist(result_path)
+
+    if show:
+        show_img = img.copy()
+        if gt_bboxes is not None:
+            show_img = draw_bbox(
+                gt_bboxes, show_img, proj_mat, img_metas, color=(255, 102, 61))
```

Comment:
We may make the color settable arguments.

Reply:
Done.